demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -93,7 +93,8 @@
     "from tensorflow.keras.layers import Dense\n",
```

Comment:
<p>It's a bit peculiar to me that this "parametrises" the text by <code>n_repeats</code>,  but mentions <code>fold = 10</code>  literally. Maybe this could be:</p><p> </p><blockquote>for repeated <code>fold</code>-fold cross-validation; that is we do <code>fold</code>-fold validation&nbsp; <code>n_repeats</code>&nbsp;times [...] train and evaluate <code>fold</code>  graph classification models</blockquote><p> </p><p>Other options could be:</p><p> </p><blockquote>for repeated 10-fold cross-validation; that is we do 10-fold validation&nbsp;5&nbsp;times [...] train and evaluate 10 graph classification models</blockquote><p> </p><blockquote>for repeated 10-fold cross-validation (set by <code>fold = 10</code> ); that is we do 10-fold validation&nbsp;5&nbsp;times (set by <code>n_repeats = 5</code> ) [...] train and evaluate 10 graph classification models</blockquote><p> </p><p>This also applies to "Finally, we plot a histogram of the accuracy of all&nbsp;<code>n_repeats x 10</code>&nbsp;models trained" below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1387/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
It's just that 10 folds is standard for cross validation but `n_repeats` is not so I think the user can set this to a value they find suitable especially if they need speed by setting it to a smaller value.